src/Dijkstra/AST/Either.agda

```diff
@@ -36,7 +36,7 @@ ASTOps.SubRet EitherOps  = EitherSubRet
 
 EitherAST = AST EitherOps
 
-module Syntax where
+module EitherSyntax where
```

Comment:
I notice you changed `MaybeSyntax` and `RWSSyntax` also.
For consistency, in a new PR, we should update `RWS` to `RWSAST` and `runRWSAST` etc

Reply:
Will do.